charts/astronomer/templates/extra-objects.yaml

```diff
@@ -0,0 +1,4 @@
+{{ range .Values.extraObjects }}
```

Comment:
This file should have a header. `helm template --show-only` returns an error if the template has null output. We sometimes do this in tests, which triggers the problem.